src/averageAges.js

```diff
@@ -7,19 +7,26 @@
  * function calculates average age only for men who died in this century
  *
  * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
+ * Divide year of personson's death by 100: Math.ceil(personson.died / 100)
  *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
+
+function averageAge(persons) {
+  return persons.reduce((acc, { died, born }) => (
+    acc + (died - born)), 0) / persons.length;
+};
+
```

Comment:
- use proper naming. Name of the function usually starts with verb that describes kind of action it is used for
- It looks clearer this way, you should pay better attention to the checklist. Code readability really matters
```suggestion
function getAverageAge(persons) {
  return persons.reduce((acc, { died, born }) => (
    acc + (died - born)
  ), 0) / persons.length;
};

```